lib/perl/Genome/Site/TGI/Command/SyncSysUserWithLdap.pm

```diff
@@ -52,40 +54,36 @@ sub execute {
 }
 
 sub get_ldap_users {
-    my $ldap_user = {};
-    my $cmd = 'ldapsearch -z 0 -x';
-    my $fh  = IO::File->new($cmd . "|");
-    my @users;
-    my $user;
-
-    # go through each line of ldapsearch output
-    while (my $c = $fh->getline) {
-        next if $c =~ /^\#/;
-        chomp($c);
-
-        if ($c =~ /^$/) {
-            # process/destroy user
-            push @users, $user;
-            undef $user;
-        } else {
-            my ($key, $value) = split(/\:\s+/,$c);
-            $user->{$key} = $value;
-        }
+    my $ldap = Net::LDAP->new('ipa1.gsc.wustl.edu', version => 3);
+    my $mesg = $ldap->start_tls(verify => 'none');
+    $mesg->code && die $mesg->error;
+
+    $mesg = $ldap->bind;
+    $mesg->code && die Dumper($mesg);
+
+    $mesg = $ldap->search(
+        base => "dc=gsc,dc=wustl,dc=edu",
+        filter => "(objectClass=Person)",
+        #filter => "(&(objectClass=Person)(uid=$username))",
+    );
```

Comment:
I asked Tom what he thought about it before committing. Since it is in 'site' we decided that it was not worth worrying about. Something like this wis generally useful, and col live in Genome::Sys. Whatcha think @tmooney ?